.gitlab-ci.yml

```diff
@@ -1,86 +1,175 @@
+# Gitlab-CI Workflow
+# stages:
+#   build:
+#     - Runs on commits on master or tags that match the pattern "v[0-9]+\.[0-9]+.*$". (e.g. 1.0, v2.1rc1)
+#   deploy-staging: 
+#     - Runs on commits on master or tags that match the pattern v1.0, v2.1rc1 (continues deployment)
+#   deploy-production:
+#     - Runs on tags that match the pattern v1.0, v2.1rc1 (manual deployment)
+
 variables:
-  BACKEND_CONTAINER_REPO:         "docker.io/parity/substrate-telemetry-backend"
-  FRONTEND_CONTAINER_REPO:        "docker.io/parity/substrate-telemetry-frontend"
-  BACKEND_IMAGE_FULL_NAME:        "${BACKEND_CONTAINER_REPO}:${CI_COMMIT_SHORT_SHA}-beta"
-  FRONTEND_IMAGE_FULL_NAME:       "${FRONTEND_CONTAINER_REPO}:${CI_COMMIT_SHORT_SHA}-beta"
-  KUBE_NAMESPACE:                 "substrate-telemetry"
-  VAULT_SERVER_URL:               "https://vault.parity-mgmt-vault.parity.io"
-  VAULT_AUTH_PATH:                "gitlab-parity-io-jwt"
-  VAULT_AUTH_ROLE:                "cicd_gitlab_parity_${CI_PROJECT_NAME}"
+  # Build Variables (Mandatory)
+  CONTAINER_REPO:             ""
+  DOCKERFILE_DIRECTORY:       ""
+  
+  # Deploy Variables (Mandatory)
+  HELM_NAMESPACE:             "substrate-telemetry"
+  HELM_RELEASE_NAME:          "substrate-telemetry"
+  HELM_CHART:                 "parity/substrate-telemetry"
+  
+  # Deploy Variables (Optional)
+  HELM_REPO_NAME:             "parity"
+  HELM_REPO_URL:              "https://paritytech.github.io/helm-charts/"
+  HELM_CONFIGMAP_NAME:        "helm-custom-values"
+  HELM_CONFIGMAP_KEYNAME:     "values-parity.yaml"
+
+  # Manual Variables (Optional)
+  ## Could be used in the webconsole when triggering the pipeline manually
+  ## DO NOT SET THEM IN THIS FILE!! They've been mentioned here only for documentation purposes!
+  FORCE_DEPLOY:               ""        # boolean: true or false - triggers the deploy-production stage
+  FORCE_DOCKER_TAG:           ""        # choose an existing docker tag to be deployed (e.g. v1.2.3)
+
+  # Vault variables (Optional)
+  VAULT_SERVER_URL:           "https://vault.parity-mgmt-vault.parity.io"
+  VAULT_AUTH_PATH:            "gitlab-parity-io-jwt"
+  VAULT_AUTH_ROLE:            "cicd_gitlab_parity_${CI_PROJECT_NAME}"
+
+
+default:
+  before_script:
+    - |-
+      echo defining DOKCER_IMAGE_TAG variable
```

Comment:
```suggestion
      echo defining DOCKER_IMAGE_TAG variable
```

Reply:
i don't see any changes here.

Reply:
The "KC" has become "CK" :)

(sometimes github sucks at updating stuff, so maybe it's not showing up properly!)

Reply:
good catch 👍